src/internal/images/common.go

```diff
@@ -1,13 +0,0 @@
-package images
```

Comment:
Good refactor, this was all that remained after the multi-arch support was added and doesn't really make sense to stand on it's own now.